opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -218,6 +218,29 @@ initFromRestartFile(const RestartValue& restartValues,
     initial_step_ = false;
 }
 
+void
+BlackoilWellModelGeneric::
+prepareDeserialize(int report_step, const size_t numCells, bool handle_ms_well)
+{
+    // wells_ecl_ should only contain wells on this processor.
+    wells_ecl_ = getLocalWells(report_step);
+    this->local_parallel_well_info_ = createLocalParallelWellInfo(wells_ecl_);
+
+    this->initializeWellProdIndCalculators();
+    initializeWellPerfData();
+
+    if (! this->wells_ecl_.empty()) {
+        handle_ms_well &= anyMSWellOpenLocal();
```

Comment:
I don't see `handle_ms_well` being used anywhere here and it's passed by value so this statement does not propagate back to the caller.  Am I missing something?

Reply:
no, my bad, it's supposed to be used below in the call to init, fixed.